lib/syntax_tree/visitor/field_visitor.rb

```diff
@@ -352,15 +352,6 @@ def visit_else(node)
         end
       end
 
-      def visit_elsif(node)
-        node(node, "elsif") do
-          field("predicate", node.predicate)
-          field("statements", node.statements)
-          field("consequent", node.consequent) if node.consequent
-          comments(node)
-        end
-      end
-
```

Comment:
I don't think this file or any of its subclasses have tests, so not 100% confident in this change. It looks like the logic in the `visit_if` method is almost identical, but I might be missing some nuance here.